napari/layers/base/base.py

```diff
@@ -33,6 +33,19 @@ class Layer(KeymapProvider, ABC):
         Scale factors for the layer.
     translate : tuple of float
         Translation values for the layer.
+    rotate : float, 3-tuple of float, or n-D array.
```

Comment:
nit picking: rotate/shear/affine is also exposed as attributes in addition to parameters in base.py, may want to add in doc string. (was hoping I could add suggestion edit here but git does not allow me to touch unmodified lines 😞 )

Reply:
updated